--- belle-sip-1.5.0/CMakeLists.txt.orig
+++ belle-sip-1.5.0/CMakeLists.txt
@@ -136,15 +136,15 @@
 
 
 set(prefix ${CMAKE_INSTALL_PREFIX})
-set(exec_prefix ${prefix}/bin)
-set(libdir ${prefix}/lib)
-set(includedir ${prefix}/include)
+set(exec_prefix ${CMAKE_INSTALL_LIBDIR})
+set(libdir ${CMAKE_INSTALL_LIBDIR})
+set(includedir ${CMAKE_INSTALL_INCLUDEDIR})
 get_filename_component(antlr3c_library_path "${ANTLR3C_LIBRARIES}" PATH)
 set(LIBS_PRIVATE "-L${antlr3c_library_path} -lantlr3c")
-get_filename_component(bctoolbox_library_path "${BCTOOLBOX_CORE_LIBRARIES}" PATH)
+get_filename_component(bctoolbox_library_path "${BCTOOLBOX_LIBRARIES}" PATH)
 set(LIBS_PRIVATE "${LIBS_PRIVATE} -L${bctoolbox_library_path} -lbctoolbox")
 if(ZLIB_FOUND)
-	set(REQUIRES_PRIVATE "${REQUIRES_PRIVATE} z")
+	set(REQUIRES_PRIVATE "${REQUIRES_PRIVATE} zlib")
 endif()
 if(HAVE_LIBDL)
 	set(LIBS_PRIVATE "${LIBS_PRIVATE} -ldl")
